lib/api/core/plugins/pluginContext.js

```diff
@@ -144,10 +144,9 @@ class PluginContext {
         document: sdk.document,
         index: sdk.index,
         ms: sdk.ms,
-        realtime: new Proxy({}, {
-          get () {
-            throw new PluginImplementationError('Realtime controller is not available in plugins. You should use plugin hooks instead');
-          }
+        realtime: Object.assign({}, sdk.realtime, {
+          subscribe: () => { throw new PluginImplementationError('realtime.subscribe method is not available in plugins. You should use plugin hooks instead'); },
+          unsubscribe: () => { throw new PluginImplementationError('realtime.unsubscribe method is not available in plugins. You should use plugin hooks instead') }
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)